receiver/prometheusremotewritereceiver/receiver.go

```diff
@@ -349,7 +349,9 @@ func (prw *prometheusRemoteWriteReceiver) translateV2(_ context.Context, req *wr
 		case writev2.Metadata_METRIC_TYPE_COUNTER:
 			addNumberDatapoints(metric.Sum().DataPoints(), ls, ts)
 		case writev2.Metadata_METRIC_TYPE_HISTOGRAM:
-			addHistogramDatapoints(metric.Histogram().DataPoints(), ls, ts)
+			// Native histograms are converted to exponential histograms
+			// More: https://opentelemetry.io/blog/2023/exponential-histograms/#opentelemetry-and-prometheus
+			addHistogramDatapoints(metric.ExponentialHistogram().DataPoints(), ls, ts)
```

Comment:
I found this information reading [this document](https://docs.google.com/document/d/1VhtB_cGnuO2q_zqEMgtoaLDvJ_kFSXRXoE0Wo74JlSY/edit?tab=t.0#heading=h.rfz4dxlnfnia). Probably this reference should not to be here(because its a blog post, not the official documentation about prometheus -> oltp). Just exposing it here to validate if I'm right or not.

Reply:
I will remove it since we already confirmed that Native should be converted to exponential